src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3394,31 +3405,33 @@ void ViewProviderSketch::setEditViewer(Gui::View3DInventorViewer* viewer, int Mo
     else
         editSubName.resize(dot - editSubName.c_str() + 1);
 
-    Base::Placement plm = getEditingPlacement();
-    Base::Rotation tmp(plm.getRotation());
+    if (AlignView.getValue()) {
+        Base::Placement plm = getEditingPlacement();
+        Base::Rotation tmp(plm.getRotation());
 
-    SbRotation rot((float)tmp[0], (float)tmp[1], (float)tmp[2], (float)tmp[3]);
+        SbRotation rot((float)tmp[0], (float)tmp[1], (float)tmp[2], (float)tmp[3]);
```

Comment:
```suggestion
        SbRotation rot = Base::convertTo<SbRotation>(plm.getRotation());
```

Should work I think? Would be good to make this small refactor as you are at it already.